tests/test_hypersphere.py

```diff
@@ -432,6 +433,15 @@ def test_dist_pairwise(self):
 
         self.assertAllClose(result, expected, rtol=1e-3)
 
+    def test_dist_pairwise_parallel(self):
```

Comment:
You do not have any test decorator here. Does that mean that we can remove `@geomstats.tests.np_and_pytorch_only` from the above `test_dist_pairwise`?

Reply:
Yes!